src/pages/nft/class/_classId/index.vue

```diff
@@ -664,34 +715,106 @@ export default {
         });
       });
       schemas.push(bookSchema);
+      if (this.nftEdition?.price) {
+        const e = this.nftEdition;
+        [
+          {
+            hid: 'og:price:amount',
+            property: 'og:price:amount',
+            content: e.price,
+          },
+          {
+            hid: 'product:price:amount',
+            property: 'product:price:amount',
+            content: e.price,
+          },
+          {
+            hid: 'og:price:currency',
+            property: 'og:price:currency',
+            content: 'USD',
+          },
+          {
+            hid: 'product:price:currency',
+            property: 'product:price:currency',
+            content: 'USD',
+          },
+          {
+            hid: 'og:availability',
+            property: 'og:availability',
+            content: e.stock ? 'in stock' : 'out of stock',
+          },
+          {
+            hid: 'product:brand',
+            property: 'product:brand',
+            content: 'NFT Book',
+          },
+          {
+            hide: 'product:locale',
+            property: 'product:locale',
+            content: this.$i18n.locale,
+          },
+          {
+            hid: 'product:catalog_id',
+            property: 'product:catalog_id',
+            content: `${this.classId}-${e.index}`,
+          },
+          {
+            hid: 'product:retailer_item_id',
+            property: 'product:retailer_item_id',
+            content: `${this.classId}-${e.index}`,
+          },
+          {
+            hid: 'product:category',
+            property: 'product:category',
+            content: 543542,
+          },
+          {
+            hid: 'product:condition',
+            property: 'product:condition',
+            content: 'new',
+          },
+          {
+            hid: 'product:custom_label_0',
+            property: 'product:custom_label_0',
+            content: this.iscnOwner,
+          },
+        ].forEach(m => meta.push(m));
+        meta.find(m => m.hid === 'og:url').content = `${EXTERNAL_HOST}${
+          this.$route.path
+        }?price_index=${e.index}`;
+        if (e.name) {
+          let titleWithEdition =
+            this.NFTName || this.$t('nft_details_page_title');
+          if (e.name) titleWithEdition += ` - ${e.name}`;
```

Comment:
`e.name` is already checked in the external conditions

```suggestion
       titleWithEdition += ` - ${e.name}`;
```